generators/client/templates/angular/src/main/webapp/app/shared/login/login.component.html.ejs

```diff
@@ -40,7 +40,7 @@
                     <input type="password" class="form-control" name="password" id="password" placeholder="{{'login.form.password.placeholder' | translate}}"
                            formControlName="password">
                 </div>
-                <div class="form-check">
+                <div class="form-check"<% if (authenticationType === 'session' && databaseType === 'no') { %> hidden<% } %>>
```

Comment:
is this related to `skipUserManagement`?

Reply:
it's more related to `no database`.